backend/controller/recipieCreator.controller.js

```diff
@@ -14,7 +14,10 @@ var exec =  promisify(require('child_process').exec)
  const CreateRecipie = async (req, res) => {
     try {
         const ingredients = req.body.foodItems ? req.body.foodItems.join(" ") : "tomatoes pasta meat"
-        exec("/Users/maryem/Documents/Archive/CreateSearchIndex " + `${ingredients}`, function(error, stdout, stderr) { 
+        exec("../backend/Archive/CreateSearchIndex " + `${ingredients}`, function(error, stdout, stderr) { 
```

Comment:
just nneds to be hardcoded. On my machine that is
`/home/ubuntu/Team-14-Bell-app/CreateSearchIndex/bin/Debug/net6.0/CreateSearchIndex`.

I guess we also use Debug instead of Release folder